addons/website_slides/static/src/js/slides_course_quiz_question_form.js

```diff
@@ -123,6 +124,18 @@ var QuestionFormWidget = publicWidget.Widget.extend({
         });
     },
 
+    /**
+     * Handler when a user clicks on the 'Save & New' button.
+     * @param ev
+     * @private
+     */
+    _validateAndCreateNewQuestion: function () {
+        var parentContext = this.getParent();
```

Comment:
No need to do this. Just initialize `this.parent` in init. And use it here. Can you give it a try?

Reply:
I gave it a try but it does not seem to work. We get `undefined` when we initialize the variable inside the `init` function.

Reply:
@shut-odoo Talking about this.parent = parent in init.